### PR TITLE
feat: Support Parachain -> Ethereum transfers

### PR DESCRIPTION
**Pull request by NunoAlexandre**

### CODE REVIEW

File: app/src/hooks/useAssetTransferApi.tsx [hunk 1]
```diff
@@ -32,11 +32,11 @@ const useAssetTransferApi = () => {
     try {
       console.log('Sender is ', JSON.stringify(sender))
 
-      //todo(nuno): get the wss and specName from the source chain
-      const { api, safeXcmVersion } = await constructApiPromise(
-        'wss://rococo-asset-hub-rpc.polkadot.io',
-      )
-      const atApi = new AssetTransferApi(api, 'asset-hub-rococo', safeXcmVersion)
+      if (!sourceChain.rpcConnection || !sourceChain.specName)
+        throw new Error('Source chain is missing rpcConnection or specName')
```
Comment (NunoAlexandre) [inline app/src/hooks/useAssetTransferApi.tsx:42]: We don't want to throw this error down to the user. Ideally, we capture it with Sentry and provide a more generic error message to the user.

Comment (noahjoeris) [reply app/src/hooks/useAssetTransferApi.tsx:42]: We do that no? We throw the error inside a try/catch. The user gets a generic error message and it is captured in sentry.

**Review by Victor-Salomon — APPROVED** [review]
What a worrrk 😍! 


There are still several comments and logs remainings. 
I'm pretty sure you added them for a reason to handle them later. 

Amazing job guys!

File: app/src/config/registry.ts [hunk 2]
```diff
@@ -335,3 +400,7 @@ export function getNativeToken(chain: Chain): Token {
       throw Error('The impossible has happened!')
   }
 }
+
+export function getRoute(env: Environment, from: Chain, to: Chain): Route | undefined {
```
[the remark this answers is not in the document]
Comment (noahjoeris) [reply app/src/config/registry.ts:404]: Yes I changed that in the other PR 👍

**Pull request merged**